src/h5web/providers/hsds/utils.ts

```diff
@@ -149,6 +150,13 @@ export function convertHsdsType(hsdsType: HsdsType): HDF5Type {
   }
 }
 
+export function convertHsdsAttributes(attrs: HDF5Attribute[]): Attribute[] {
+  return attrs.map((attr) => ({
+    ...attr,
+    shape: convertHsdsShape(attr.shape),
```

Comment:
Actually, do the `type` of attribute need to be converted as well ?

The type `HDF5Attribute[]` might be wrong here.

Reply:
I think you're right. It probably comes from:

```ts
export type HsdsAttributeWithValueResponse = HDF5Attribute;
```

Reply:
Fixed in my other PR. Will merge in here once you confirm it's all good: https://github.com/silx-kit/h5web/pull/589/commits/52933a8c05342e4f51613d8505b6a19c66809d76